### ISSUE TITLE
French translation

### ISSUE THREAD

**Issue by utybo** [steverichey/DMLocalizer#10]
```
A game by: Un jeu par
Powered by: Powered by
Press left or right: Appuyez sur la touche droite ou gauche
Release left and right to start: Relâchez les touches droite et gauche pour démarrer
Release left and right to erase: Relâchez les touches droite et gauche pour effacer
Fill bar to erase save data: Remplissez la barre pour effacer les données de sauvegarde
Erasing save data: Effacement des données de sauvegarde en cours
attempts: essais
distance: distance
seconds: secondes
medals: médailles
levels: niveaux
coins: pièces
trophies: trophées
zenith: zénith
zeniths: zéniths
the end: the end
trophy unlocked: trophée débloqué
paused: en pause
time: temps
Hold left or right to restart: Maintenez la touche droite ou gauche pour redémarrer
Don't Move: Don't Move
Level Up: Niveau Sup.
Game Over: Game Over
```

Notes : Any French person can understand Game Over and any French translation would ruin this text.
There is not any translation for Powered by. If you find "propulsé par", please note that this translation is only applicable to vehicles. It does not make sense to use propulsé par with a video game in French.
Niveau sup. means Niveau supérieur (=Level up, did not think of this translation when I first made this issue...)

Images :
![zenith_la](https://cloud.githubusercontent.com/assets/5175705/5057545/a18e3432-6cc6-11e4-99af-cae0c670655e.png)
![levelup_la](https://cloud.githubusercontent.com/assets/5175705/5988465/582d39fa-a961-11e4-98dd-710425668fb6.png)

EDIT : fixed typo
EDIT 2 : Added another translation

**Comment by steverichey** [comment]
Thank you!

**Comment by utybo** [comment]
Fixed typo again, everything should be ok now

**Comment by steverichey** [comment]
@utybo, how would you like to be credited?

**Comment by utybo** [comment]
Sorry, I did not see your comment...
Well, if you could just put "utybo" somewhere in the game and a link to my website (http://utybo.github.io) that would be perfect !
BTW, I need to modify 2 or 3 things, I will edit this comment when it's done

EDIT : Done! Sorry again for the late comment :/

**Comment by steverichey** [comment]
Thanks!

**Comment by steverichey** [comment]
Implemented in `e5dcd11b`
